docs/playbook/getting-set-up/using-github.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Using GitHub
+description: "A guide on how to carry out common tasks in the GitHub UI"
+layout: playbook_js
+tags: [playbook]
+---
+
```

Comment:
Maybe add a prerequisite of having a Gtihub account & how to get one, think this is on the introduction page at the moment but I think it makes more sense here